package.json

```diff
@@ -238,6 +238,11 @@
                         "w"
                     ]
                 },
+                "vscode-neovim.leaveInsertModeOnEscape": {
```

Comment:
Not in favor of another random option, especially one that now needs to be checked in various places.

 Is there not a way to achieve this with vscode keyboard shortcut settings? We could then document that.

Reply:
Not sure I can figure out which other places this option has to be checked.
Currently, dismissing a VSCode autocomplete/suggestion menu with Escape would enter normal mode: this doesn't seem right to me.
Trying to tweak VSCode keyboard shortcut settings wasn't successful and, even if possible, my feeling is that it would be too brittle

```json
  {
    "key": "escape",
    "command": "noop",
    "when": "textInputFocus && !inlineSuggestionVisible"
  },
```